### PR TITLE
Add README.

### PR DESCRIPTION
**Pull request by nevinheintze**
Replace the placeholder README.md with a full README.
Includes a not-complete-yet warning at the top of the file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,442 @@
+***WARNING: HarbourBridge is currently under development, and not all modules
+have been checked in yet. In short, it probably won't compile right now. Please
```
Comment (lgruen) [inline README.md]: checked in ==> merged?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,442 @@
...
+[Files Generated by HarbourBridge](#files-generated-by-harbourbridge). Details
+of how PostgreSQL is mapped to Spanner can be found in the
+[Schema Conversion](#schema-conversion) section.
+
+This tool is part of the Cloud Spanner Ecosystem, a community contributed and
+supported open-source repository. Please [report
+issues](https://github.com/cloudspannerecosystem/harbourbridge/issues) and send
+pull requests. Note that this tool is not officially supported as part of the
+Cloud Spanner product.
+
```
Comment (lgruen) [inline README.md]: Do you want to link to the whitepaper here?

File: README.md (hunk 1 above)
Comment (nevinheintze) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (nevinheintze) [reply README.md]: I'll add link when I add the whitepaper.

**Pull request merged**